docs/contact.md

```diff
@@ -1,14 +1,18 @@
 ---
 title: &title Contact
-description: &description Looking to use the WebContainer API for more expansive use cases? Get in touch by filling out the form.
+description: &description Contact the WebContainer API team
 head:
   - ['meta', {property: 'og:title', content: *title}]
   - ['meta', {property: 'og:image', content: 'https://webcontainers.io/img/og/contact.png'}]
   - ['meta', {name: 'twitter:title', content: *title}]
   - ['meta', {name: 'twitter:description', content: *description}]
 ---
-# Contact
+# Contact Us
 
-Looking to use the WebContainer API for more expansive use cases?
+For general inquiries you can reach out to us on X [@StackBlitz](https://x.com/stackblitz) or email [hello@stackblitz.com](mailto:hello@stackblitz.com) both of which are actively monitored.
```

Comment:
```suggestion
For general inquiries you can reach out to us on X [@StackBlitz](https://x.com/stackblitz) or email [hello@stackblitz.com](mailto:hello@stackblitz.com).
```

Us reading the channels is a given as since we've suggested them, a redundant assurance tends to [decrease trust](https://medium.com/@Cindy_Rosen/never-trust-a-man-who-says-trust-me-d1a448ab17f8)